templates/migrate/daml.yaml.template

```diff
@@ -0,0 +1,13 @@
+sdk-version: __VERSION__
+name: __PROJECT_NAME__
+source: daml/Main.daml
+parties:
+  - Alice
+  - Bob
+version: 1.0.0
+exposed-modules:
```

Comment:
Probably better to remove the `exposed-modules` section -- `daml build` will generate it automatically.

Reply:
makes sense!